pgml-apps/pgml-chat/pyproject.toml

```diff
@@ -11,14 +11,21 @@ packages = [{include = "pgml_chat"}]
 python = ">=3.8,<4.0"
 openai = "^0.27.8"
 rich = "^13.4.2"
-pgml = "^0.9.0"
+pgml = {version =  "0.9.4", source = "testpypi"}
```

Comment:
should we wait for a release?

Reply:
I'm planning to push pgml-chat to pypi after pgml 0.9.4 is released to pypi. In the meantime, want to update master.